### PR TITLE
Update /desktop/developers what’s new in 17.04 copy

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Update /desktop/developers What’s new section as per copy doc

## QA

- Pull code and run ./run
- Go to http://localhost:8001/desktop/developers
- Check that the What’s new section is the same as copy doc
- Make sure the laptop image has gone as per card

## Issue / Card

Doc: https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#heading=h.78m14zr2rvaj

Card: https://trello.com/c/F0j0GPmJ

### CODE REVIEW

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -78,15 +78,23 @@
...
+                  <li class="last-item">Print without printer-specific drivers on selected printers</li>
+                </ul>
                 <ul class="list-ticks six-col last-col">
-                    <li>Unity 8 desktop interface (Ubuntu developer preview)</li>
-                    <li>Selected Unity 8 apps included: Ubuntu Browser, System Settings and Terminal</li>
-                    <li class="last-item">Install and run snaps from the Terminal</li>
+                  <li>Print without printer-specific drivers on selected printers</li>
+                  <li>Most GNOME applications updated to 3.24</li>
+                  <li>Linux 4.10 kernel</li>
+                  <li>New installations use swap file instead of swap partition</li>
```
Comment (anthonydillon) [inline templates/desktop/developers.html]: last-item?

File: templates/desktop/developers.html [hunk 2]
```diff
@@ -78,15 +78,23 @@
...
+                <ul class="list-ticks six-col">
+                  <li>LibreOffice 5.3 with new UI for improved usability</li>
+                  <li>Fast install snaps with snap URL Support added to Ubuntu Software 3.22</li>
+                  <li class="last-item">Print without printer-specific drivers on selected printers</li>
+                </ul>
                 <ul class="list-ticks six-col last-col">
-                    <li>Unity 8 desktop interface (Ubuntu developer preview)</li>
-                    <li>Selected Unity 8 apps included: Ubuntu Browser, System Settings and Terminal</li>
-                    <li class="last-item">Install and run snaps from the Terminal</li>
+                  <li>Print without printer-specific drivers on selected printers</li>
```
Comment (anthonydillon) [inline templates/desktop/developers.html]: You have this item twice.

**Comment by anthonydillon** [comment]
@grahambancroft You will need design QA instructions.

File: templates/desktop/developers.html (hunk 2 above)
Comment (grahambancroft) [inline templates/desktop/developers.html]: Fixed

File: templates/desktop/developers.html (hunk 1 above)
Comment (grahambancroft) [inline templates/desktop/developers.html]: The last-item class is only required on the last item of the first list when in a combined-list container, so on small it adds a border, however, this seems to be broken site-wide. I’ll add a separate issue.

**Comment by grahambancroft** [comment]
@anthonydillon I shouldn’t have added the design needed label as this will be reviewed in the 17.04 walk through in the morning

**Pull request merged**